component/file_cache/file_cache_test.go

```diff
@@ -1377,52 +1369,45 @@ func (suite *fileCacheTestSuite) TestRenameFileAndCacheCleanup() {
 	suite.assert.NoFileExists(suite.fake_storage_path + "/" + src) // Src does not exist
 	suite.assert.FileExists(suite.fake_storage_path + "/" + dst)   // Dst does exist
 
-	suite.fileCache.CloseFile(internal.CloseFileOptions{Handle: openHandle})
-
-	time.Sleep(1 * time.Second)                           // Check once before the cache cleanup that file exists
+	time.Sleep(500 * time.Millisecond)                    // Check once before the cache cleanup that file exists
 	suite.assert.FileExists(suite.cache_path + "/" + dst) // Dst shall exists in cache
 
-	time.Sleep(2 * time.Second)                           // Wait for the cache cleanup to occur
+	time.Sleep(1 * time.Second)                           // Wait for the cache cleanup to occur
 	suite.assert.FileExists(suite.cache_path + "/" + dst) // Dst shall not exists in cache
 }
 
 func (suite *fileCacheTestSuite) TestRenameFileAndCacheCleanupWithNoTimeout() {
 	defer suite.cleanupTest()
-	suite.cleanupTest()
-
-	config := fmt.Sprintf("file_cache:\n  path: %s\n  offload-io: true\n  timeout-sec: 0\n\nloopbackfs:\n  path: %s",
-		suite.cache_path, suite.fake_storage_path)
-	suite.setupTestHelper(config) // setup a new file cache with a custom config (teardown will occur after the test as usual)
 
 	src := "source5"
 	dst := "destination5"
-	createHandle, _ := suite.fileCache.CreateFile(internal.CreateFileOptions{Name: src, Mode: 0666})
-	openHandle, _ := suite.fileCache.OpenFile(internal.OpenFileOptions{Name: src, Mode: 0666})
-	suite.fileCache.CloseFile(internal.CloseFileOptions{Handle: createHandle})
+	handle, _ := suite.fileCache.CreateFile(internal.CreateFileOptions{Name: src, Mode: 0666})
+	suite.fileCache.CloseFile(internal.CloseFileOptions{Handle: handle})
 
-	// Path should be in the file cache
-	suite.assert.FileExists(suite.cache_path + "/" + src)
-	// Path should be in fake storage
+	// Path _might_ be in the file cache
+	// Path _should_ be in fake storage
 	suite.assert.FileExists(suite.fake_storage_path + "/" + src)
 
 	// RenameFile
 	err := suite.fileCache.RenameFile(internal.RenameFileOptions{Src: src, Dst: dst})
 	suite.assert.NoError(err)
-	// Path in fake storage and file cache should be updated
-	suite.assert.NoFileExists(suite.cache_path + "/" + src)        // Src does not exist
-	suite.assert.FileExists(suite.cache_path + "/" + dst)          // Dst shall exists in cache
+
+	// Dst _might_ exist in cache, and the rename should be complete in fake storage
 	suite.assert.NoFileExists(suite.fake_storage_path + "/" + src) // Src does not exist
 	suite.assert.FileExists(suite.fake_storage_path + "/" + dst)   // Dst does exist
 
-	suite.fileCache.CloseFile(internal.CloseFileOptions{Handle: openHandle})
-
-	time.Sleep(100 * time.Millisecond)                      // Wait for the cache cleanup to occur
-	suite.assert.NoFileExists(suite.cache_path + "/" + dst) // Dst shall not exists in cache
+	time.Sleep(100 * time.Millisecond) // Wait for the cache cleanup to occur
```

Comment:
I no longer see the config being defined with the `timeout-sec`. is there a default i'm missing?

Reply:
The default config set in `SetupTest()` [on line 101](https://github.com/Seagate/cloudfuse/blob/minor-improvements-to-filecache/component/file_cache/file_cache_test.go#L101) is identical to the custom config that was used in this test. So I just dropped the custom config here since it was redundant.